### PR TITLE
DO NOT MERGE Testing CI if it works with this change

### PR DESCRIPTION
**Pull request by omBratteng**
#3277 keeps failing on tests with this change. Just want to test if its because of external user, or if it's this change.

### Preview domain
https://ombratteng-patch-1.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/account/notifications.tsx [hunk 1]
```diff
@@ -274,7 +274,11 @@ const AccountNotificationsPage = (): ReactElement => {
       });
     }
 
-    subscribePersonalizedDigest({ type, hour: preferredHour });
+    subscribePersonalizedDigest({
+      type,
+      hour: preferredHour,
+      sendType: personalizedDigestType as SendType,
+    });
```
Comment (omBratteng) [inline packages/webapp/pages/account/notifications.tsx:281]:
```suggestion
    subscribePersonalizedDigest({ type, hour: preferredHour });
    // comment for change
```

**Comment by sshanzel** [comment]
I think it is due to the change. Some parameters could be missing from the GraphQL mock.

File: packages/webapp/pages/account/notifications.tsx [hunk 2]
```diff
@@ -275,6 +275,7 @@ const AccountNotificationsPage = (): ReactElement => {
     }
 
     subscribePersonalizedDigest({ type, hour: preferredHour });
```
Comment (omBratteng) [inline packages/webapp/pages/account/notifications.tsx:277]:
```suggestion
    subscribePersonalizedDigest({
      type,
      hour: preferredHour,
      sendType: personalizedDigestType,
    });
```

**Comment by sshanzel** [comment]
Yeah, should be fixed now. I tried to pull some test changes on a different branch.

**Comment by omBratteng** [comment]
> I think it is due to the change. Some parameters could be missing from the GraphQL mock.

Could be. The jest errors doesn't say what test fail (I think, didn't look through it all)

**Comment by omBratteng** [comment]
I did test that it works in the preview deployment

**Pull request closed without merging**